### PR TITLE
Article: JavaScript do...while loop

### PR DESCRIPTION
**Pull request by nag786**
created an article on how to use do..while loop

### CODE REVIEW

**Comment by theflametrooper** [comment]
@nag786 Please update your PR description...

File: do-while.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# do...while Loop
+-------------
+
+The do...while loop is very related to while loop. In the do while loop, the condition is checked at the end of the loop.
```
Comment (alayek) [inline do-while.md]: > The ~~do...while~~`do-while` loop is ~~very~~ closely related to `while` loop.

File: do-while.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# do...while Loop
```
Comment (alayek) [inline do-while.md]: > # ~~do...while~~`do-while` loop

File: do-while.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+-------------
+
+The do...while loop is very related to while loop. In the do while loop, the condition is checked at the end of the loop.
+
+Here is the **syntax** for do...while loop:
```
Comment (alayek) [inline do-while.md]: > ~~do...while~~`do-while`

File: do-while.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+________
+
+### Syntax:
+
```
Comment (alayek) [inline do-while.md]: Put syntax inside code block snippet

```js
do {
   // Statements
}
while(condition);
```

File: do-while.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+ {
+   
+   *Statement(s);*
+   
+}
+while (*condition*);
+
+
+
+**statement(s):** A statement that is executed at least once before the condition or Boolean expression is evaluated and is re-executed each time the condition evaluates to true.
```
Comment (alayek) [inline do-while.md]: highlight the part about _at least once_.

File: do-while.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+**statement(s):** A statement that is executed at least once before the condition or Boolean expression is evaluated and is re-executed each time the condition evaluates to true. 
+
+**condition:** Here, a condition is a Boolean expression. If Boolean expression evaluates to true, the statement is executed again. When Boolean expression evaluates to false, the loops ends.
+
+______
+
+
+### **Example:**
+    var i = 0;
```
Comment (alayek) [inline do-while.md]: Put code and output inside code block. Basically put anything related to code inside multi-line or inline code syntax.

File: do-while.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    1 
+    2 
+    3 
+    4
+    5
+	
+   ___
+   
+   source:[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/do...while)
+
```
Comment (alayek) [inline do-while.md]: Remove all the blank lines at the end of it.

File: do-while.md (hunk 1 above)
Comment (abhisekp) [inline do-while.md]: It's [**do...while**](https://devdocs.io/javascript/statements/do...while) loop

File: do-while.md (hunk 2 above)
Comment (abhisekp) [inline do-while.md]: # `do...while` Loop 

```md
# `do...while` Loop 
```
is fine

File: do-while.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
+# do...while Loop
+-------------
+
+The do...while loop is very related to while loop. In the do while loop, the condition is checked at the end of the loop.
+
```
Comment (abhisekp) [inline do-while.md]: Here add a heading 2 for the love of @camperbot like this

```md
## (add a space after ##)
```

**Comment by abhisekp** [comment]
**File Rename:-**
```diff
- do-while.md
+ JS-do-while-loop.md
```

File: do-while.md (hunk 6 above)
Comment (abhisekp) [inline do-while.md]: @nag786 and also use proper indentation. Use [**JS Beautify**](http://jsbeautifier.org) to auto beautify the code.

**Comment by Rafase282** [comment]
See https://github.com/FreeCodeCamp/wiki/pull/795

**Pull request closed without merging**